test/run-k8s-integration.sh

```diff
@@ -13,6 +13,7 @@ readonly PKGDIR=${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
 readonly overlay_name="${GCE_PD_OVERLAY_NAME:-prow-head-template}"
 readonly boskos_resource_type="${GCE_PD_BOSKOS_RESOURCE_TYPE:-gce-project}"
 readonly do_driver_build="${GCE_PD_DO_DRIVER_BUILD:-true}"
+export GCE_PD_VERBOSITY=9
```

Comment:
should this also be configureable?

Reply:
for integration tests we always want to just set it to max. noisy logs for our tests is fine and helps with debugging